pkgs/by-name/f3/f3d/package.nix

```
@@ -17,6 +20,7 @@
   fontconfig,
   withManual ? !stdenv.hostPlatform.isDarwin,
   withPythonBinding ? false,
+  withUsd ? true,
```

```suggestion
  withUsd ? openusd.meta.available,
```

considering it [takes 1h to build](https://hydra.nixos.org/build/297077604) it is no fun when it is broken. I have a vague memory of it being broken for some stretch of time as well.

not a blocker

---

oh wow, that's quite clever! is applied